### ISSUE TITLE
error persisting piece: entrygeterror,entrygeterror

### ISSUE THREAD

**Issue by TimoP21** [kzahel/jstorrent-legacy-app#150]
When downloading a torrent, everything seems to be working for about a minute and then I get an error such as error persisting piece: entrygeterror,entrygeterror . Any idea why?

Thanks

**Comment by kzahel** [comment]
Try going to app settings and changing your download directory and then remove the torrent and try again.

**Comment by TimoP21** [comment]
this did actually work. Originally the destination folder was an SD card, I changed it to the internal memory and the error no longer occurred. 

Strange though as the SD card has plenty of room on it and previously worked until I had a large queue of torrents to download when the error first happened.

**Comment by kzahel** [comment]
Yep, that can happen to SD cards because the write speed is typically slower than chromebook internal storage. If your download is faster than the SD card write speed, this can sometimes cause a problem.

**Comment by kzahel** [comment]
The legacy Chrome App is no longer maintained. I've rebuilt JSTorrent from scratch — the new version is ready at **[jstorrent.com](https://jstorrent.com)**

If you run into any issues, please leave feedback!
